src/SDKs/_metadata/managedservices_resource-manager.txt

```diff
@@ -0,0 +1,14 @@
+﻿Installing AutoRest version: latest
+AutoRest installed successfully.
+Commencing code generation
+Generating CSharp code
```

Comment:
Please delete this file and regenerate the code, commit the new file.

Reply:
will do thanks.